renderer/containers/action-bar.js

```diff
@@ -19,26 +20,28 @@ export default class ActionBarContainer extends Container {
   }
 
   setDisplay = display => {
-    const {width, height} = display;
+    const {width, height, cropper} = display;
+    const {x, y, ratioLocked} = cropper ? this.settings.get('actionbar') : {};
 
     this.setState({
       screenWidth: width,
       screenHeight: height,
-      x: (width - 464) / 2,
-      y: Math.ceil(height * 0.8),
-      width: 464,
-      height: 64
+      x: x || (width - barWidth) / 2,
```

Comment:
Are you sure you didn't want to use a ternary here? `||` will use `(width - barWidth) / 2` also when `x` is `0`.